site/docs/basics/ajv.zh-CN.md

```
@@ -0,0 +1,444 @@
+---
+title: 参数校验
+---
+
+# Ajv 入参校验
+
+参考 [egg-typebox-validate](https://github.com/eggjs-community/egg-typebox-validate) 的最佳实践，结合 ajv + [typebox](https://github.com/sinclairzx81/typebox?tab=readme-ov-file#json-types) + [ErrorCode]() 统一错误码规范，只需要定义一次参数校验 Schema，就能同时拥有参数校验和类型定义（完整的 TypeScript 类型提示）。
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)